app/components/pages/AuthorGuide/EditorialProcess.js

```diff
@@ -0,0 +1,313 @@
+      the evaluation of my submission to eLife?
+    </H3>
+    <Paragraph>
+      eLife does offer &quot;scoop protection&quot; in the sense that your
+      initial submission will not be rejected on the grounds that it lacks
+      novelty because of the paper in the other journal. However, it is still
+      possible that the initial submission will be rejected on other grounds.
+    </Paragraph>
+
+    <H3>
+      5. If I make an initial submission to eLife and a preprint on a similar
+      topic is posted on a recognized server before a decision has been made on
+      my initial submission, will the preprint influence the evaluation of my
+      submission to eLife?
+    </H3>
+    <Paragraph>
+      As in the previous case, your initial submission will not be rejected on
+      the grounds that it lacks novelty because of the preprint. However, it is
+      still possible that the initial submission will be rejected on other
+      grounds.
+    </Paragraph>
+
+    <H3>
+      6. If I make an initial submission to eLife and a manuscript on a similar
+      topic is already being reviewed by eLife, or has been accepted for
+      publication by eLife, will the other manuscript influence the evaluation
+      of my submission to eLife?
+    </H3>
+    <Paragraph>
+      As in the previous case, your initial submission will not be rejected on
+      the grounds that it lacks novelty because of the other manuscript.
+      However, it is still possible that the initial submission will be rejected
+      on other grounds.
+    </Paragraph>
+
+    <H3>
+      7. If my initial submission to eLife is available as a preprint and it
+      attracts negative/critical feedback (eg, in the form of comments and/or
+      other preprints) before a decision has been made, will this feedback
+      influence the evaluation of my submission?
+    </H3>
+    <Paragraph>
+      The Editors will primarily base their decision on the content of your
+      submission, but public feedback on the work (positive or negative) might
+      be taken into account. If a full submission is invited, you are advised to
+      be open about the existence of such feedback in your cover letter.
+    </Paragraph>
+
+    <H3>8. Can I cite a preprint in my submission to eLife?</H3>
+    <Paragraph>
+      Yes, providing it has been posted on a recognised public preprint server
+      and has a persistent ID. Preprints should be cited as follows: -
+      Author(s). Year. Title. Name of preprint server. doi or url For example: -
+      Narasimhan VM et al. 2016. A direct multi-generational estimate of the
+      human mutation rate from autozygous segments seen in thousands of
+      parentally related individuals. bioRxiv. doi:
+      https://doi.org/10.1101/059436 - Chung J, Gulcehre C, Cho K, Bengio Y.
+      2014. Empirical evaluation of gated recurrent neural networks on sequence
+      modeling. arXiv. http://arxiv.abs/1412.3555
```

Comment:
In the original, this paragraph is presented as a list but here it gets flattened into one block of text.